app/views/users/edit.html.erb

```diff
@@ -0,0 +1,59 @@
+<h1> Edit Form</h1>
+<!--
+<form action=" <%= users_path %> " method="post" accept-charset="UTF-8">
+<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+<label for="username">Username</label>
+<input type="text" name="user[username]"  id="user_username"/><br>
+<label for="email">Email</label>
+<input type="email" name="user[email]" id="user_email"/><br>
+<label for="password">Password</label>
+<input type="password" name="user[password]" id="user_password"/><br>
+<input type="submit" value="submit"/>
+</form>
```

Comment:
- Use "users_path" instead of " <%= users_path %> " in action
- Add this field `
  <input
    type="hidden"
    name="authenticity_token"
    value="<%= form_authenticity_token %>"
  />
`

Reply:
Noted and thanks for the feed 👍👍.